pipeline.py

```diff
@@ -0,0 +1,100 @@
+import data_discovery_ai.utils.preprocessor as preprocessor
+import data_discovery_ai.model.keywordModel as model
+import data_discovery_ai.utils.es_connector as connector
+import data_discovery_ai.service.keywordClassifier as keywordClassifier
+import numpy as np
+import json
+import pandas as pd
+
+
+def pipeline():
```

Comment:
I think you should need to make this pipeline a class instead of tight coupled sequence under one single giant method, each of the step is a separate method with meaningful names and type hinting and description, e.g step 0 is init() 

when you break the steps to method, try to make it not too broad or atomic small unnecessarily, group related actions together by nature or method signature etc.

then you can have a main method that executes sub methods in order. It will be easier to read and debug/maintain later on. Or even if one day we decide to orchestrate the pipeline with tools such as Prefect, we can easily wrap the sub-methods with `@task` decorator and the main method with `@flow` decorator etc.

do you familiar with writing unit tests? ideally you need test cases to cover each of the step that returning values.

Reply:
i don't like seeing too much code being commented out.

Reply:
> I think you should need to make this pipeline a class instead of tight coupled sequence under one single giant method, each of the step is a separate method with meaningful names and type hinting and description, e.g step 0 is init()
> 
> when you break the steps to method, try to make it not too broad or atomic small unnecessarily, group related actions together by nature or method signature etc.
> 
> then you can have a main method that executes sub methods in order. It will be easier to read and debug/maintain later on. Or even if one day we decide to orchestrate the pipeline with tools such as Prefect, we can easily wrap the sub-methods with `@task` decorator and the main method with `@flow` decorator etc.
> 
> do you familiar with writing unit tests? ideally you need test cases to cover each of the step that returning values.

convert the functions within a class. Please let me know if these works for API use or not.
I'd like to separate the unit test to another PR. Have several tasks to do so will work on the unit test when I have time.


> i don't like seeing too much code being commented out.

Me too, just want to make sure these code can run from your side so provide an example of use.